portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +33,53 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String[] COMMENTS = {
-    "Wow good job on the first portfolio website",
-    "You should find some help, the design is wacky",
-    "Where am I?"
-  };
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertArrayToJson(COMMENTS);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    String json = getDataStoreAll(datastore);
+
     response.setContentType("text/html;");
     response.getWriter().println(json);
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity commentsEntity = new Entity("Comments");
```

Comment:
nit: let's move this line to the line above commentsEntity.setProperty("name", name)

Reply:
What do you mean by this?

Reply:
you declare commentsEntity on line 49, but you don't do anything with it until line 57
it's ok as is, but it might be easier to to reader if you to declare `commentsEntity` right above line 57 where it's actually used

Reply:
ok, done